packages/caliper-cli/caliper.js

```diff
@@ -16,24 +16,28 @@
 'use strict';
 
 process.env.SUPPRESS_NO_CONFIG_WARNING = true;
+const { CaliperUtils } = require('@hyperledger/caliper-core');
+const Logger = CaliperUtils.getLogger('cli');
 const yargs = require('yargs');
 const version = 'v' + require('./package.json').version;
 
 let results = yargs
     .commandDir('./lib')
     .help()
-    .example('caliper bind\ncaliper benchmark run\ncaliper zooclient start\ncaliper zooservice start  ')
+    .example('caliper bind\ncaliper benchmark run\n caliper worker launch')
     .demand(1)
     .wrap(null)
     .strict()
```

Comment:
This can be removed, or the `strict(false)` should be moved here

Reply:
good spot!!